scripts/final_test.py

```
@@ -0,0 +1,225 @@
+        
+        # Test retrieving contact
+        contact = contact_manager.get_contact("Test Contact")
+        assert contact is not None, "Failed to retrieve contact"
+        print("✅ Contact retrieved")
+        
+        # Test message processor
+        processor = MessageProcessor(db_manager)
+        print("✅ Message processor created")
+        
+        # Test export manager
+        export_manager = ExportManager(db_manager, processor)
```

Ruff (0.14.0)

Local variable `export_manager` is assigned to but never used

Remove assignment to unused variable `export_manager`

(F841)